src/Stratis.SmartContracts.IntegrationTests/ContractPoAStressTests.cs

```diff
@@ -48,8 +48,7 @@ public void MaximumCreateTransactionsInABlock()
             this.mockChain.MineBlocks(1);
 
             // Just over block gas limit.
-            // TODO: Update with block gas limit changes
-            const int expectedTxsInBlock = 85;
+            const int expectedTxsInBlock = 84;
             var lastBlock = node1.GetLastBlock();
             Assert.Equal(expectedTxsInBlock, lastBlock.Transactions.Count);
```

Comment:
```suggestion
            Assert.Equal(expectedTxsInBlock, lastBlock.Transactions.Count);
            
           const int expectedInMempool = txsToSend - expectedTxsInBlock + 1; // Left in mempool. Total - all in block, except for coinbase.
            Assert.Equal(expectedInMempool, node1.CoreNode.FullNode.MempoolManager().InfoAll().Count);
```

Reply:
Just one last check I think we should add to the test. I was worried that there might be some logic in the coinview rules that was removing transactions from the mempool when they weren't included in the SC code.